.github/actions/build-dependencies-installation/action.yml

```diff
@@ -25,6 +25,7 @@ runs:
       pkg-config
 
   - name: Set up Python 3
+    if: ${{ inputs.toolchain == 'emscripten' || inputs.toolchain == 'pnacl' }}
```

Comment:
ESLint also needs Python

Reply:
Also `coverage` as well, because we build the whole `smart_card_connector_app` in all modes except ASan:

https://github.com/GoogleChromeLabs/chromeos_smart_card_connector/blob/0ba4103b135d3a622a2550917507d1852ba63a3c/Makefile#L82-L92

and the latter triggers:

https://github.com/GoogleChromeLabs/chromeos_smart_card_connector/blob/0ba4103b135d3a622a2550917507d1852ba63a3c/smart_card_connector_app/build/Makefile#L115

(I might fix this some day, because conceptually `coverage` only needs the C++ code to be built, but right now we just build everything.)

So I guess right now we can only really exclude `asan_testing`?..

Reply:
i'll close it, since we decided to go with https://github.com/GoogleChromeLabs/chromeos_smart_card_connector/pull/1021 instead.